dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java

```diff
@@ -2132,9 +2140,12 @@ private boolean isCharEncodingSet()
                 return isCharEncodingExplicit;
             }
         }
+        // if contentType is null, it means no setContentType has called before but setCharacterEncoding may have called
+        // so check isCharEncodingExplicit and return;  Not sure why only return if ContentType is set ?
```

Comment:
`Not sure why only return if ContentType is set?` Is this still an outstanding question we need to answer? Can we clean up the comment?